pos_x0: 0.008
pos_y0 : -4.168
pos_z0: 0.0 # Must be 0.0 for the kobuki

orient_x0: 0.0
orient_y0: 0.0
orient_z0: 0.0 # 0.72
orient_w0: 0.1 # 0.68

num_persons: 3